lxd-agent/tests/test_rss_calculation.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eu
+
+echo "=== RSS Calculation Validation Test ==="
+
+# Run free command to get baseline
+FREE_USED=$(free | grep "^Mem:" | awk '{print $3}')
```

Comment:
`grep` isn't strictly required as `awk` can do filtering on its own:

```suggestion
FREE_USED=$(free | awk '/^Mem:/ {print $3}')
```

Just an FYI, no need to change.

Reply:
Thank you for pointing that out.

Reply:
You've got the same pattern elsewhere in the script if you feel like dropping `grep` ;)

Reply:
I will do that.  It makes the code look cleaner for sure.